lib/link.js

```diff
@@ -143,12 +144,16 @@ class Link extends BaseCommand {
       log: this.npm.log,
       add: names.map(l => `file:${resolve(globalTop, 'node_modules', l)}`),
       save,
+      workspaces: this.workspaces,
     })
 
     await reifyFinish(this.npm, localArb)
   }
 
   async linkPkg () {
+    const wsp = this.workspacePaths
+    const paths = wsp && wsp.length ? wsp : [this.npm.prefix]
```

Comment:
Everywhere else I've seen localPrefix used, when dealing with workspaces.  I don't know if this is wrong specifically here, but I do know we need to either say "yes it's ok here" or not, and then revisit how we think about the prefix/localPrefix/globalPrefix because it seems prone to misuse.

Reply:
Link throws in global mode so this particular case is ok